src/Microsoft.Tye.Core/StandardOptions.cs

```diff
@@ -46,16 +46,16 @@ public static Option Tags
             }
         }
 
-        public static Option Force
-        {
-            get
-            {
-                return new Option("--force", "Force overwrite of existing files")
-                {
-                    Argument = new Argument<bool>(),
-                };
-            }
-        }
+        public static Option Framework =>
+                    new Option(new string[] { "-f", "--framework" })
+                    {
+                        Description = "The target framework to run for. The target framework must also be specified in the project file.",
```

Comment:
We should mention that this option is only applied to multi-targeted projects.

Reply:
We should also mention that this option overrides the TFM specified in the tye.yaml for multi-targeted projects. Essentially we should be clear that this option is used as the TFM for all multi-targeted projects in the tye-yaml specification.

Reply:
you mean chaing the `Description` ?
I'm usually very bad for such thing, If you have any idea internally (I mean with the maintaining team, feel free to push an update on that part)

Reply:
Okay I'll write up something.